src/common/format/common.fbs

```diff
@@ -89,6 +97,8 @@ table TaskReply {
   state: long;
   // A local scheduler ID.
   local_scheduler_id: string;
+  // A string of bytes representing the task's TaskExecutionDependencies.
+  execution_dependencies: string;
```

Comment:
wouldn't it be cleaner to store the list of strings here directly instead of wrapping and serializing them? Any reason this is not possible?